README.md

```diff
@@ -99,6 +100,31 @@ module.exports = {
 }
 ```
 
+### [Transpile](http://coffeescript.org/#transpilation)

```

Comment:
Could you do my a favour and wrap all headings in backticks please ?  😛  `\### [Transpile]()` => ```\### [`Transpile`]()```

Reply:
Okay.